Tests/Unit/NetworkServer/LoRaDeviceFactoryTest.cs

```diff
@@ -136,6 +141,12 @@ protected override LoRaDevice CreateDevice(IoTHubDeviceInfo deviceInfo)
                 LastDeviceMock = device;
                 return device.Object;
             }
+
+            public override ILoRaDeviceClient CreateDeviceClient(string deviceId, string primaryKey)
+            {
+                LastDeviceClientMock = new Mock<ILoRaDeviceClient>();
+                return LastDeviceClientMock.Object;
+            }
```

Comment:
```suggestion
            public override ILoRaDeviceClient CreateDeviceClient(string deviceId, string primaryKey) => Mock.Of<ILoRaDeviceClient>();
```

Reply:
I need to assign the Mock to the LastDeviceClientMock in order to do some verifications on it.
I could condense it to the following, but I am not sure if it's still readable enough 
```suggestion
                        public override ILoRaDeviceClient CreateDeviceClient(string deviceId, string primaryKey) => (LastDeviceClientMock = new Mock<ILoRaDeviceClient>()).Object;
```

Reply:
sorry, overread the fact that it is assigned to a member. no worries then.